docs/source/llms/tracing/tracing-schema.rst

```diff
@@ -331,3 +331,45 @@ documents from a vector store). The ``RETRIEVER`` span type has the following sc
 
 \* For example, both ``[Document(page_content="Hello world", metadata={"doc_uri": "https://example.com"})]`` and
 ``[{"page_content": "Hello world", "metadata": {"doc_uri": "https://example.com"}}]`` are valid outputs for a ``RETRIEVER`` span.
+
+Chat Completion Spans
```

Comment:
Can we add a quick example of adding those attributes to a manually created span? I think it makes it more clearer that we use custom attributes here not the actual input/output.